package.json

```diff
@@ -56,6 +56,7 @@
     "babel-jest": "^21.2.0",
     "babel-loader": "7.1.2",
     "babel-plugin-flow-runtime": "0.11.1",
+    "babel-plugin-syntax-async-functions": "^6.13.0",
```

Comment:
Does not babel do it out of the box already?  `babel-plugin-syntax-async-functions` . I think after all merges we have to review the policy of presets